### ISSUE TITLE
Accessibility: Organization options menu is not keyboard accessible

### ISSUE THREAD

**Issue by patrickhlauke** [bitwarden/clients#3180]
### Steps To Reproduce

1. Navigate using the keyboard (`Tab`, `Shift+Tab`, `Enter`, `Space`, cursor keys)
2. In the left-hand filter section, set focus on a "..." options control for an organization
3. Open the options menu

### Expected Result

As a keyboard user, I should be able to either have the option items in the menu focusable (using `Tab`) - which would make this menu more of a disclosure widget - or be able to use cursor keys to cycle/move through the menu items - making this more of an app-like menu,

### Actual Result

While the options menu can be opened, the menu items ("Leave" in this case) can't be focused nor activated using the keyboard.

### Screenshots or Videos

![Screenshot showing an organization, a focused three-dot options menu button, and the open menu showing the 'Leave' option](https://user-images.githubusercontent.com/895831/180894603-902ffe0b-0000-4c92-8b20-25b056e5cad4.png)

### Operating System

Windows

### Operating System Version

10 Build 19044

### Web Browser

Chrome

### Browser Version

101.0.4951.54 (Official Build) (64-bit)

### Build Version

2022.6.2

**Comment by dbosompem** [comment]
Hi @patrickhlauke , thank you for raising this. We will take a look at it and make time to resolve it.

**Comment by patrickhlauke** [comment]
I plan, once I get a moment, to do a proper review of some of the filters stuff on the website ... so there might even be a chance you'll see a PR come your way to tackle this as well :)
https://github.com/bitwarden/clients/pull/2689#issuecomment-1194785138

**Comment by dbosompem** [comment]
Great. Thank you, @patrickhlauke, for your willingness to assist every time. Very much appreciated!

**Comment by bnagawiecki** [comment]
Hey @patrickhlauke,

Thanks as always for submitting these.

I got this replicated and will raise internally for prioritization.

**Comment by Greenderella** [comment]
Hi @patrickhlauke, it appears that this issue is no longer reproducible. 

![navigation](https://github.com/user-attachments/assets/33aed57b-ded2-400d-b774-a12724478937)

I can interact with the three dots and select either Yes or No to exit an organization using the keyboard. If there's anything I might have missed, please let me know; otherwise, I'll proceed to close this issue.

**Comment by patrickhlauke** [comment]
ah, it's been ages since i used the actual web interface (rather than desktop app or extension). confirming that the menu and its "Leave..." item can now be properly operated using the keyboard.
